pyccel/codegen/codegen.py

```diff
@@ -126,28 +117,22 @@ def modules(self):
 
     @property
     def is_module(self):
-        """Returns True if a Module."""
+        """Returns True if a Program."""
```

Comment:
hmm, I don't why you changed that.

Reply:
XD I deleted this function then put it back with copy paste, that's why it shows up as a stupid change

Reply:
Fixed 5041be14